### ISSUE TITLE
Code review

### ISSUE THREAD

**Issue by kgryte** [dstructs/typed-array-reviver#1]
Dependent on dstructs/typed-array-to-json#1 due to dev dependency.

**Comment by kgryte** [comment]
Will also require a `dstructs-array` dependency bump (possibly `1.x.x`), due to recent work on the `array` API.

**Comment by Planeshifter** [comment]
LGTM. One suggestion: In the README.md file, maybe it should be clarified that the JSON string has to be created via the `typed-array-to-json` module. As of now, this becomes clear only by looking at the example code.

**Comment by kgryte** [comment]
Agreed. Did something similar with [utils-error-reviver](https://github.com/kgryte/utils-error-reviver). Will add when I upgrade `dstructs-array`.
